docs/BUILD

```diff
@@ -35,23 +35,32 @@ genrule(
 )
 
 genrule(
-    name = "lexicon_html",
-    srcs = deps + [
+    name = "lexicon_content",
+    srcs = [
         "lexicon.html",
         "lexicon_entry.html",
-        "template.html",
         ":rules",
     ],
+    outs = ["lexicon_content.html"],
+    cmd = [
+        '"$TOOL" -i docs/lexicon.html -i docs/lexicon_entry.html docs/rules.json > "$OUT"',
+    ],
+    tools = ["//docs/tools/lexicon_templater"],
+)
+
+genrule(
+    name = "lexicon_html",
+    srcs = deps + [
+        ":lexicon_content",
+        "template.html",
+    ],
     outs = ["lexicon.html"],
     cmd = [
-        '"$TOOLS_LEX" -i docs/lexicon.html -i docs/lexicon_entry.html docs/rules.json > "$OUT"',
-        '"$TOOLS_TMPL" --template docs/template.html --in lexicon.html > tmp.html',
+        'mv docs/lexicon_content.html "$OUT"',
```

Comment:
Why?

And more accurately, why not:

```
genrule(
    name = "lexicon_html",
    srcs = deps + [
        ":lexicon_content",
        "template.html",
    ],
    outs = ["lexicon.html"],
    cmd = [
        '"$TOOL" --template docs/template.html --in docs/lexicon_content.html > "$OUT"',
    ],
    tools = ["//docs/tools/templater"],
    visibility = ["//docs/test/..."],
)
```

similarly below.

I'm not a fan of how the command relies on filenames rather than using `$SRCS`, but I'm guessing the template implicitly depends on the files in `deps`, which makes it hard to use the dict-style `srcs`